common/ux-window.cpp

```diff
@@ -215,13 +216,20 @@ namespace rs2
    
     void ux_window::open_window()
     {
+        IMGUI_CHECKVERSION();
+        ImGui::CreateContext();
+        ImGuiIO& io = ImGui::GetIO();
+        io.ConfigFlags |= ImGuiConfigFlags_NavEnableKeyboard;     // Enable Keyboard Controls
+        io.ConfigFlags |= ImGuiConfigFlags_NoMouseCursorChange;
```

Comment:
Don't we want a cursor change?
Please check metadata scroll works

Reply:
Was added in order to solve the flickering cursor we got on hover (flag changes accordingly). 
Checked and the metadata scroll works

Reply:
Great, please ass a comment saying why this flag was added and what does it mean

Reply:
added comment